content/blog/introducing-pulumi-import-command/index.md

```diff
@@ -0,0 +1,432 @@
+---
+title: "Introducing The Pulumi Import Command"
```

Comment:
```suggestion
title: "Automatically Create Infrastructure as Code by Importing Cloud Resources"
```

I'd suggest something that's appealing to a slightly broader audience than marketing to our existing users.

Reply:
👍 
ok :)